controllers/virtualmachinepublishrequest/virtualmachinepublishrequest_controller_unit_test.go

```diff
@@ -718,14 +774,159 @@ func unitTestsReconcile() {
 						})
 					})
 				})
+
+				When("library type is inventory", func() {
+					When("task is nil", func() {
+						BeforeEach(func() {
+							itemID = "vm-123"
+							clv1a2 := builder.DummyContentLibraryV1A2("dummy-cl", vm.Namespace, "dummy-id")
+
+							initObjects = append(initObjects, clv1a2)
+						})
+						JustBeforeEach(func() {
+							pkgcfg.SetContext(ctx, func(config *pkgcfg.Config) {
+								config.Features.InventoryContentLibrary = true
+							})
+
+							fakeVMProvider.GetCloneTasksForVMFn = func(
+								ctx context.Context,
+								vm *vmopv1.VirtualMachine,
+								moID, descriptionID string) ([]vimtypes.TaskInfo, error) {
+
+								return nil, nil
+							}
+							fakeVMProvider.GetItemFromInventoryByNameFn = func(
```

Comment:
This could be removed as well

Reply:
No, this function is used in more than one place. The path that is being exercised here requires that the returned value is not nil

Reply:
IC. Thanks